app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_assigned_list.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.ReassignCaseController
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{AssignedCases, Case}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@(cases: Option[AssignedCases])(implicit request: AuthenticatedRequest[_], messages: Messages)
+
+@if(cases.isDefined) {
```

Comment:
This kind of thing should be tested in a View test

Reply:
You are correct.  I'm missing view specs to assert this type of logic.  Good spot